### PR TITLE
Fix Android unknown notification permission

### PR DESCRIPTION
**Pull request by Zazo032**
As stated here: https://github.com/Vanethos/flutter_notification_permissions/issues/15#issuecomment-589001782

### CODE REVIEW

File: android/src/main/java/com/vanethos/notification_permissions/NotificationPermissionsPlugin.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           result.error(call.method, "context is not instance of Activity", null);
         }
       } else {
-        result.success(null);
+        result.success(PERMISSION_GRANTED);
```
Comment (Zazo032) [inline android/src/main/java/com/vanethos/notification_permissions/NotificationPermissionsPlugin.java]: This was returning always null (`unknown`) when it should return `granted`. The same happens when it's `denied`.

Comment (Vanethos) [reply android/src/main/java/com/vanethos/notification_permissions/NotificationPermissionsPlugin.java]: Yikes, how did I miss this? thank you so much :pray:

**Comment by Vanethos** [comment]
@Zazo032 this PR was marked as Draft, but from what I've seen from the code it's working at this moment.

Can I merge the PR?

**Comment by Zazo032** [comment]
Yes! I don't have experience with iOS code, can you check if this was happening too in iOS?

**Comment by Vanethos** [comment]
As far as I can see it, it seems to be working

the offending piece of code:

```swift
func getNotificationStatus(completion: @escaping ((String) -> Void)) {
      if #available(iOS 10.0, *) {
          let current = UNUserNotificationCenter.current()
          current.getNotificationSettings(completionHandler: { settings in
              if settings.authorizationStatus == .notDetermined {
                  completion(self.permissionUnknown)
              } else if settings.authorizationStatus == .denied {
                  completion(self.permissionDenied)
              } else if settings.authorizationStatus == .authorized {
                  completion(self.permissionGranted)
              }
          })
      } else {
          // Fallback on earlier versions
          if UIApplication.shared.isRegisteredForRemoteNotifications {
              completion(self.permissionGranted)
          } else {
              completion(self.permissionDenied)
          }
      }
  }
```

(pasted it here because you might be interested since swift and kotlin are quite similar)

Can I merge it? 👍 

If there's a problem with iOS, we'll open a separate issue

**Comment by Zazo032** [comment]
Yes! You can merge it. Swift code seems like it should work properly. Thanks!

**Pull request merged**